packages/map-template/src/components/MapsIndoorsMap/MapsIndoorsMap.jsx

```diff
@@ -110,6 +112,7 @@ function MapsIndoorsMap(props) {
 			category: props.supportsUrlParameters && categoryQueryParameter ? categoryQueryParameter : props.category,
             searchAllVenues: props.supportsUrlParameters && searchAllVenuesParameter ? searchAllVenuesParameter : (props.searchAllVenues || defaultProps.searchAllVenues),
             hideNonMatches: props.supportsUrlParameters && hideNonMatchesQueryParameter ? hideNonMatchesQueryParameter : props.hideNonMatches,
+            showExternalIDs: props.supportsUrlParameters && showExternalIDsQueryParameter ? showExternalIDsQueryParameter : props.showExternalIDs,
```

Comment:
What is `showExternalIDsQueryParameter` is false?

Reply:
Then externalId is not shown when we click on the location.